internal/driver/webui.go

```diff
@@ -264,6 +265,7 @@ func (ui *webInterface) render(w http.ResponseWriter, tmpl string,
 	data.Title = file + " " + profile
 	data.Errors = errList
 	data.Total = rpt.Total()
+	data.SampleTypes = sampleTypes(rpt.Profile())
```

Comment:
Can you just use ui.prof?

Reply:
Yes and I actually had this written that way at first. I switched to this approach because `ui.prof` was used to generate the `rpt`, which is the input to this method, so it felt appropriate to look at the profile from `rpt`. This also avoids leaking the abstraction that `ui.prof == rpt.prof`.

Happy to switch it back if you'd like, I don't have strong feelings.

Reply:
Adding an accessor for the profile in the report is more leaking since it changes the report public API surface, while here we are just dealing with local context of a single file implementation, so yes, please change. There isn't much motivation for passing the report object here, BTW, so we could as well just pass the total which is the only thing the render function uses.

Reply:
Done.